.github/workflows/app_customers_deployment_dev.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       TRANSALL_UPLOAD_KEYSTORE_JKS_BASE64: ${{ secrets.TRANSALL_UPLOAD_KEYSTORE_JKS_BASE64 }}
       GOOGLE_PLAY_STORE_CREDENTIALS_BASE64: ${{ secrets.GOOGLE_PLAY_STORE_CREDENTIALS_BASE64 }}
       TRANSALL_FIREBASE_CLI_TOKEN: ${{ secrets.TRANSALL_FIREBASE_CLI_TOKEN }}
-      APP_BASE_ADDRESS: ${{ secrets.APP_BASE_ADDRESS_DEV }}
+      APP_BASE_ADDRESS: "https://transtu-dev.karibu-cap.com"
```

Comment:
maybe we can define some variables to make that more manageable

Reply:
each env defined the host. so instead of put it inside secret for each env, i don't understand your request.